packages/manager/modules/pci/src/components/project/gateways/gateways-model-selector/template.html

```diff
@@ -18,13 +18,13 @@
                 data-ng-if="gateway.blobs.technical.bandwidth.level"
             >
                 <strong
-                    data-translate="{{ 'pci_projects_project_gateways_model_selector_bandwidth'}}"
-                    data-translate-values="{ bandwidth : gateway.blobs.technical.bandwidth.level }"
+                    data-translate="{{ 'pci_projects_project_public_gateways_add_spec_' + gateway.product  + '_bandwidth'}}"
```

Comment:
Since we have same text for all the sizes, it has been modified to use same translation key for all the sizes. So use `pci_projects_project_gateways_model_selector_bandwidth`.